python/hail/matrixtable.py

```diff
@@ -1577,9 +1631,14 @@ def transmute_entries(self, **named_exprs):
         Returns
         -------
         :class:`.MatrixTable`
-            Annotated matrix table.
         """
-        raise NotImplementedError()
+        caller = 'MatrixTable.transmute_entries'
+        e = get_annotate_exprs(caller, named_exprs, self._entry_indices)
+        fields_referenced = extract_refs_by_indices(e.values(), self._entry_indices)
```

Comment:
Can you combine these into a single line? Same for all 5 instances.

Reply:
combine which? The set and set subtraction?

Reply:
Yes.